Userland/Libraries/LibWeb/Layout/TableFormattingContext.cpp

```diff
@@ -468,6 +468,118 @@ void TableFormattingContext::determine_intrisic_size_of_table_container(Availabl
     }
 }
 
+void TableFormattingContext::compute_row_measures(AvailableSpace const& available_space)
```

Comment:
could we replace `compute_column_measures` and `compute_row_measures` with one function that calculate measures depending on specified direction?

Reply:
I've tried it, with templates. Unfortunately, it requires around 7-8 helpers to abstract differences away, in the end it didn't reduce the line count and didn't look like a clear win. I'm agnostic about it, if you really want it that way I can rework it.

Reply:
I went a simpler way in GridFormattingContext where parameter that indicates whether rows and columns are processed is passed as a regular argument. you could try that too if you want :)

Reply:
The main problem is the multiplexing of logic, not how we route it. I've unified them, in any case.